test/integration_test/kubevirt_test.go

```diff
@@ -246,3 +216,87 @@ func createDatadiskTemplates(t *testing.T) error {
 	}
 	return nil
 }
+
+func kubevirtVMScaledDeployAndValidate(t *testing.T, instanceID string, appKeys []string, scale int) []*scheduler.Context {
+	var allCtxs []*scheduler.Context
+	logrus.Infof("Deploying %d VMs and validating them.", scale)
+	for i := 1; i <= scale; i++ {
+		logrus.Infof("Deploying VM: %d", i)
```

Comment:
since we are deploying multiple VMs in each iteration (one per appKey), we could use a better message here
e.g. `Deploying %d VMs in the iteration %d of %d", len(apKeys), i, scale`